### PR TITLE
Mailer copy update  - References did not come back in time

### PR DESCRIPTION
**Pull request by TobyRet**
## Context

If references haven't come back by 18 Sep the candidate gets an email 18 Sep at 11:59pm.

## Changes proposed in this pull request

- Update copy in mailer
- Rename references => ‘application_on_pause’ is now ‘referees_did_not_respond_before_end_of_cycle’. This is a better description in light of the copy changes

## Guidance to review

[Example of updated notification](https://www.notifications.service.gov.uk/services/022acc23-c40a-4077-bbd6-fc98b2155534/notification/11d651c1-0875-4ea8-946f-84696846167b)

Note that subject has since been updated to correct wording.

## Link to Trello card

https://trello.com/c/KUyyYMxJ/2132-%F0%9F%9A%B2%F0%9F%94%9A-references-didnt-come-back-in-time-%F0%9F%8F%88

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] API release notes have been updated if necessary
- [ ] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/views/candidate_mailer/application_on_pause.erb [hunk 1]
```diff
@@ -1,21 +1,15 @@
 Dear <%= @application_form.first_name %>,
```
Comment (tvararu) [inline app/views/candidate_mailer/application_on_pause.erb:1]: `def referees_did_not_respond` for this email?

**Review by tvararu — APPROVED** [review]

**Comment by EmmaFrith** [comment]
thanks @TobyRet 

Just realised we haven't introduced the link with an action and a colon as we normally do - can you make a small edit: 

You can update your referees and apply again for courses starting in the 2021 to 2022 academic year.

Apply again:

https://apply-for-te-2131-refer-9arwvk.herokuapp.com/candidate/confirm_authentication?token=suw1SiPVxBV_UJzzcF--&u=bnZKenhRcHU0Ykd6TUM5UjV5VVBEZz09LS12OWRFVmhUT3VnWkp4OGZ4N0J4Uld3PT0%3D--f17377d5bacadb0367ddb643b00c5c2ee87ce2b5

File: app/views/candidate_mailer/application_on_pause.erb (hunk 1 above)
Comment (TobyRet) [reply app/views/candidate_mailer/application_on_pause.erb]: `referees_did_not_respond_before_end_of_cycle` following discussion with @EmmaFrith

**Comment by EmmaFrith** [comment]
thanks @TobyRet - just to confirm, this will take the candidate to their banner right? 

Apply again:

<%= @candidate_magic_link %>

**Pull request merged**